ArduPlane/Plane.h

```diff
@@ -261,6 +264,9 @@ class Plane : public AP_HAL::HAL::Callbacks {
     // selected navigation controller
     AP_SpdHgtControl *SpdHgt_Controller = &TECS_controller;
 
+    //Soaring Controller
+    SoaringController soaring_controller {ahrs, SpdHgt_Controller, aparm, &DataFlash, LOG_THERMAL_MSG, LOG_VARIO_MSG};
```

Comment:
Thanks, I wasn't aware of this. Removed the DataFlash parameter in PR #5737. The other two macros are needed as parameters though, because libraries/SoaringController is unaware of macros defined in ArduPlane/Log.h, and shouldn't be.